lib/context_adapter_helper.js

```diff
@@ -23,7 +23,9 @@
 
 'use strict';
 
+var caLogger = require('logops');
 var caConfig = require('./context_adapter_configuration');
+caLogger.setLevel(caConfig.LOGOPS_LEVEL);
```

Comment:
Fixed in https://github.com/telefonicaid/context-adapter/commit/e4eb9b44a5ba1749c70061332463f31fd3298869